WickedEngine/wiHelper.cpp

```diff
@@ -382,7 +382,13 @@ namespace wiHelper
 	bool FileRead(const std::string& fileName, std::vector<uint8_t>& data)
 	{
 #ifndef PLATFORM_UWP
+#ifdef SDL_FILESYSTEM_UNIX
+		std::string unixPath = fileName;
+		std::replace(unixPath.begin(), unixPath.end(), '\\', '/');
+		ifstream file(unixPath, ios::binary | ios::ate);
+#else
```

Comment:
do we really need the else part? we can just do the replace in the `#ifdef SDL_FILESYSTEM_UNIX` section and load the file anyway